GitCommands/AsyncLoader.cs

```diff
@@ -21,7 +21,7 @@ public sealed class AsyncLoader : IDisposable
         /// <param name="continueWith">An action to invoke on the main thread with the return value from <paramref name="loadContent"/>.</param>
         /// <param name="onError">
         /// A callback for notification of exceptions from <paramref name="loadContent"/>.
-        /// Invoked on the original synchronisation context.
+        /// Invoked on the original synchronization context.
```

Comment:
Why have you changed this? UK and Australian English use "s" not "z" 😉

Reply:
It's because I trusted ReSharper's dictionary )

Reply:
I would prefer if such changes are avoid in the future: 1) this isn't relevant to the change and 2) it is not a typo/incorrect spelling

Reply:
> It's because I trusted ReSharper's dictionary

I'm using an en-GB dictionary, though R# defaults to en-US even on en-GB machines. If you see things like this then it's simplest to add an exception for such words to the `GitExtensions.sln.DotSettings` file. There are a bunch in there already. You can do it from the lightbulb menu (something like "add to team shared dictionary").

BTW just tried this PR out and it looks very slick. Good work.

Reply:
thx 😃